run-tests.js

```diff
@@ -239,15 +250,19 @@ async function main() {
   // concurrent ones
   for (const test of nonConcurrentTestNames) {
     let passed = false
-
+    console.log(test)
```

Comment:
Should this be removed?

Reply:
Yeah removed in https://github.com/vercel/next.js/pull/27604/commits/9b9170d6b8f4607d9fa68c540b5ac289a5c5cf58